packages/local_auth/local_auth/android/src/test/java/io/flutter/plugins/localauth/LocalAuthTest.java

```diff
@@ -20,45 +23,83 @@
 import io.flutter.embedding.engine.plugins.lifecycle.HiddenLifecycleReference;
 import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.MethodChannel;
+import java.util.Collections;
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mock;
 
 public class LocalAuthTest {
-  @Test
-  public void isDeviceSupportedReturnsFalse() {
-    final LocalAuthPlugin plugin = new LocalAuthPlugin();
-    final MethodChannel.Result mockResult = mock(MethodChannel.Result.class);
-    plugin.onMethodCall(new MethodCall("isDeviceSupported", null), mockResult);
-    verify(mockResult).success(false);
-  }
 
-  @Test
-  public void onDetachedFromActivity_ShouldReleaseActivity() {
-    final Activity mockActivity = mock(Activity.class);
-    final ActivityPluginBinding mockActivityBinding = mock(ActivityPluginBinding.class);
-    when(mockActivityBinding.getActivity()).thenReturn(mockActivity);
+  private LocalAuthPlugin plugin = new LocalAuthPlugin();
```

Comment:
The object under test should not be part of the test fixture; that's an anti-pattern in unit testing.